src/bench/verify_script.cpp

```diff
@@ -4,7 +4,7 @@
 
 #include <bench/bench.h>
 #include <key.h>
-#if defined(HAVE_CONSENSUS_LIB)
+#if HAVE_CONSENSUS_LIB
```

Comment:
This is not defined when `./configure --with-libs=no`. So it will default to `0` or something?